content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,182 @@
+
+- Aggregation of news in the XMPP universe
+- Short formulation of news and events
+- Summary of the monthly communication on extensions (XEP)
+- Review of the draft
+- Translations: especially French, German and Spanish
+
+The drafting process for each newsletter [is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- [German](https://www.jabber.de/)
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+## Software releases
+
+It has been almost a year since the release of Gajim 1.1.3. A year of developing new features, cleaning up old code, and fixing bugs. This month, it is finally the time for a [first beta release of Gajim 1.2](https://gajim.org/post/2020-05-08-gajim-1.2.0-beta1-released/). Highlights are (amongst others): improved group chat system, completely rewritten network code, and a new account creation assistant. But there is much more to discover.
+
+### Clients and applications
+
+[Gajim Development News May 2020](https://gajim.org/post/2020-05-26-development-news-may/): New account badges and the great plugin overhaul. In preparation of the upcoming 1.2 release, many plugins have been revisited. Check out the new file preview!
+
+Goffi released [Salut à Toi progress note for calendar week 21](https://www.goffi.org/b/kmebCba3m38dr9sgzgGrkp/progress-note)!
+
+[Poezio](https://poez.io) released 0.13.0 and 0.13.1 which now requires python 3.7. These versions bring Poezio into the club of OMEMO supporting clients and also enable HTTP upload via dedicated plugins. Check the full [changelog](https://lab.louiz.org/poezio/poezio/-/blob/master/CHANGELOG) to look at all the improvements of these versions.
+
+[Biboumi](https://biboumi.louiz.org/), the XMPP/IRC gateway, was released in version 0.8.5, fixing compilation issues with GCC 10.
+
+The python based XMPP automatic file sender [Errol has been updated to v2.0.1](https://blog.agayon.be/errol_update1.html) which modernizes its async i/o calls and adds more privacy. By moving from 'aionotify' to the better maintained 'watchdog', Errol theoretically extends its support (from Linux only) to Windows, Mac OS X and FreeBSD.
+
+## Services
+
+blabber.im now [supports IPv6](https://blabber.im/blabber-im-ist-nun-ipv6-tauglich/). The service operators created an [invitation link which works for their server but also for every other ID](https://blabber.im/en/i-moved/). If you moved to an XMPP based chat client, you can now conveniently put this link into the status of your previous non-XMPP chat clients. This link offers your contacts all the necessary infos to be able to reach you and (if they want to) follow you into the XMPP world.
+
+jabber.org migrates its server to prosody. [Stay informed!](https://www.jabber.org/notices.html)
```

Comment:
Prosody

Reply:
I'd say a word about jabber.org like:
"The jabber.org server is quite a central place in the community and has been historic in many ways. It started with the jabberd server, went on ejabberd, then M-Link, and now it's Prodosy time. For the community coding the server, it is both a blessing and a hell: because many clients connect and load the server, and malevolent people attack it, you can then spot a lot of issues and thus massively improve the codebase."

Reply:
yes